database/migrations/2021_01_21_155342_create_notifications_table.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateNotificationsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('notifications', function (Blueprint $table) {
+            $table->id();
+            $table->string('title');
+            $table->text('body');
+            $table->timestamp('valid_from');
+            $table->timestamp('valid_to');
+            $table->timestamp('disabled_at')->nullable();
```

Comment:
Laravel has a trait for this - [SoftDeletes](https://laravel.com/docs/8.x/eloquent#soft-deleting), might be a good candidate to use here?

Reply:
Good shout, didn't think of that! 68a5d3c